st2api/st2api/controllers/v1/actions.py

```diff
@@ -235,9 +236,20 @@ def delete(self, ref_or_id, requester_user):
             ref_or_id,
             action_db,
         )
+        
+        BASE_PATH = cfg.CONF.system.base_path
+        pack_name = action_db["pack"]
+        entry_point = action_db["entry_point"]
+        metadate_file = action_db["metadata_file"]
+        ACTION_PYTHON_FILE_PATH = os.path.join(BASE_PATH, "packs", pack_name, "actions", entry_point)
+        ACTION_METADATA_FILE_PATH = os.path.join(BASE_PATH, "packs", pack_name, metadate_file)
```

Comment:
Two nitpicks:

1. Not all action entrypoints are Python files.
2. Variable names in Python don't need to be uppercased.

```suggestion
        action_entrypoint_file_path = os.path.join(BASE_PATH, "packs", pack_name, "actions", entry_point)
        action_metadata_file_path = os.path.join(BASE_PATH, "packs", pack_name, metadate_file)
```